### PR TITLE
support update-insert and update-delete event for update primary keys

### PR DESCRIPTION
**Pull request by seanzhougoogle**

### CODE REVIEW

File: src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java [hunk 1]
```diff
@@ -248,6 +249,17 @@ private void fetchNext() throws Exception {
     }
   }
 
+  private DMLOperation.Type getOperationType(String changeType) {
```
Comment (avirmani) [inline src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java:252]: Can we just add some comments here as to why we check on these event types as well.

File: src/test/java/io/cdap/delta/datastream/DatastreamEventConsumerTest.java [hunk 2]
```diff
@@ -303,6 +303,73 @@ public void testDelete() throws Exception {
...
+        //second event is an update-insert
+        assertEquals(DMLOperation.Type.INSERT, operation.getType());
+        assertNull(event.getPreviousRow());
+        assertEquals(211L, row.<Long>get("EMPLOYEE_ID"));
+        assertTrue(operation.getSizeInBytes() > 0);
+      }
+
+      assertEquals("Sean", row.<String>get("FIRST_NAME"));
+      assertEquals("Zhou", row.<String>get("LAST_NAME"));
+      assertEquals("seanzhou@google.com", row.<String>get("EMAIL"));
```
Comment (avirmani) [inline src/test/java/io/cdap/delta/datastream/DatastreamEventConsumerTest.java]: Can we use a fake account here?

**Review by avirmani — APPROVED** [review]

File: src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java (hunk 1 above)
Comment (seanzhougoogle) [reply src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java:252]: done :)

File: src/test/java/io/cdap/delta/datastream/DatastreamEventConsumerTest.java (hunk 2 above)
Comment (seanzhougoogle) [reply src/test/java/io/cdap/delta/datastream/DatastreamEventConsumerTest.java]: done:)

**Pull request merged**